src/gui/QuickStartHandler.hh

```diff
@@ -69,6 +78,9 @@ class GZ_SIM_GUI_VISIBLE QuickStartHandler : public QObject
 
   /// \brief Get starting world url.
   private: std::string startingWorld{""};
+
+  // Add a member for the transport node
+  private: std::unique_ptr<gz::transport::Node> node;
```

Comment:
I think this can just be `gz::transport::Node node` (without being a `std::unique_ptr`)